.github/workflows/rust_bors.yml

```diff
@@ -16,29 +16,25 @@ env:
 
 jobs:
   rust:
-    strategy:
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-
-    runs-on: ${{ matrix.os }}
```

Comment:
The bors workflow uses a matrix, to test marker on ubuntu, windows and macos. Bors times out, as it was waiting on the `rust (macos-latest), ...` jobs. Could you readd the matrix? Then everything should work again :)

Reply:
I see, I mistakenly removed it when copying. Will fix